pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -218,6 +221,55 @@ func (r *MigPlan) ListMigrations(client k8sclient.Client) ([]*MigMigration, erro
 	return list, nil
 }
 
+// ListTemplatePods - get list of pod templates, associated with a plan resource
+func (r *MigPlan) ListTemplatePods(client k8sclient.Client) ([]corev1.Pod, error) {
```

Comment:
I don't understand why this method is on MigPlan.  The existing `List` or `Get` convenience methods *only* pertain to resources referenced directly (or indirectly) by the Plan. Further, the functionality is solely the concern of the migration controller.

Reply:
I think an earlier iteration of this wasn't on migplan. I think this was because we need to call GetSourceNamespaces on the plan within this method. Another possibility would be to make it a top level function that takes either the migplan, or the migplan's list of src namespaces as an argument.

Reply:
Seems like it should just go in `pvlist.go` and pass in the `plan` where it's used.

Reply:
It's also used in migmigration/stage.go. I know in the past, things that are called from multiple controllers end up pushed into the apis pkg. Is it OK for it to go in migplan/pvlist.go (but as an exported function) which would then be called from stage.go as well?

Reply:
Ah, didn't realize it's used by multiple controllers.  In that case, `MigPlan` is a reasonable place for it for the reasons you mentioned.